.github/workflows/docs-and-coverage.yml

```diff
@@ -6,32 +6,6 @@ on:
       - main
 
 jobs:
-  generate-docs:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout the repository
-        uses: actions/checkout@v2
-
-      - name: Install dependencies
-        run: yarn install
-
-      - name: Create the docs
-        run: yarn run docs
-
-      - name: Collect coverage report
-        run: yarn test:coverage
-
-      - name: Create Coverage Badges
-        uses: jaywcjlove/coverage-badges-cli@e07f25709cd25486855c1ba1b26da53576ff3620
-        with:
-          source: coverage/coverage-summary.json
-          output: coverage/lcov-report/badges.svg
-
-      - name: Prepare folder for gh-pages
-        run: mv coverage/lcov-report ./docs/coverage
-
-      - name: Deploy 🚀
-        uses: JamesIves/github-pages-deploy-action@8817a56e5bfec6e2b08345c81f4d422db53a2cdc
-        with:
-          branch: gh-pages
-          folder: docs
+  generate-docs-and-coverage:
```

Comment:
Looks like we need to configure paths

Reply:
```yml
with:
      prepare-gh-pages-commands: |
        mv docs/* ./gh-pages
        mv coverage/lcov-report ./gh-pages/coverage
```

Reply:
But it's the [default value](https://github.com/fingerprintjs/dx-team-toolkit/blob/main/.github/workflows/docs-and-coverage.yml#L7) for this input, right?